content/build.ts

```diff
@@ -0,0 +1,500 @@
+/**
+ * Build Script for Navigator Content
+ *
+ * This script generates static JSON files from content sources (markdown, JSON) for use
+ * in the business.nj.gov My Account application. It processes 8 content types:
+ * industries, tasks, actions, certifications, filings, fundings, license events, and reinstatements.
+ *
+ * ## Architecture
+ *
+ * 1. **Infrastructure Layer** - File system operations (readFile, writeFile, etc.)
+ *    - Abstracted behind FileSystemPort interface for dependency injection
+ *    - Makes the code 100% testable without touching the real file system
+ *
+ * 2. **Domain Layer** - Pure business logic (parseIndustryFile, buildIndustries, etc.)
+ *    - No dependencies on file system or external libraries
+ *    - Receives file system port as parameter for all I/O operations
+ *
+ * 3. **Application Layer** - Orchestration (buildAndWriteContent, executeBuild, main)
+ *    - Coordinates domain functions with infrastructure
+ *    - Uses metaprogramming pattern (contentConfigs array) to eliminate code duplication
+ *
+ * ## Metaprogramming Pattern
+ *
+ * Instead of writing 7 nearly-identical build functions for each of the Markdown imports, we use a configuration-driven approach:
+ * - contentConfigs array defines metadata for each content type
+ * - buildAndWriteContent() generic helper handles all content types
+ * - executeBuild() loops through configs instead of calling individual functions
+ */
+
+import fs from "node:fs";
+import path from "node:path";
```

Comment:
Using the sync versions rather than the async versions of these modules. There's nothing here that would benefit from us using the asynchronous versions. It takes, like, no time to build these files.